data/json/terrain.json

```diff
@@ -3836,6 +3836,89 @@
       "items": [ { "item": "withered", "prob": 50, "count": [ 1, 2 ] } ]
     }
   },
+  {
+    "type": "terrain",
+    "id": "t_shrub_rose",
+    "name": "rose bush",
+    "description": "A fat bush of beautiful red roses, if only you could get a date!  Watch out for its thorns!",
+    "symbol": "#",
+    "color": "light_red_green",
+    "move_cost": 8,
+    "flags": [ "TRANSPARENT", "CONTAINER", "FLAMMABLE_ASH", "THIN_OBSTACLE", "SHRUB", "SHORT", "SHARP" ],
+    "transforms_into": "t_shrub_rose_harvested",
+    "examine_action": "harvest_ter_nectar",
+    "harvest_by_season": [ { "seasons": [ "autumn" ], "entries": [ { "drop": "rose_hips", "base_num": [ 2, 5 ], "scaled_num": [ 0, 0.5 ] } ] } ],
```

Comment:
I guess that is from #25360, right?

![image](https://user-images.githubusercontent.com/16213433/45268508-27bb3e00-b486-11e8-8f57-a5556b7e3003.png)

Reply:
Yeah, it's trying to drop rose hips where are the fruit after the flower and I added those in the other PR.